lte/gateway/c/core/oai/test/s1ap_task/test_s1ap_state_converter.cpp

```diff
@@ -20,7 +20,7 @@ extern "C" {
 }
 
 #include "lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_handlers.hpp"
-#include "lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_converter.hpp"
+#include "lte/gateway/c/core/oai/include/state_converter.hpp"
```

Comment:
We can just remove these test cases since we are not "converting" the state any more. All the conversion functions are replaced by `MergeFrom`, which is not a function we need to unit test here.

Reply:
Ya I thought to remove these test cases after we perform stability testing

Reply:
As discussed removing this file